pkg/infoutil/infoutil.go

```diff
@@ -151,6 +157,64 @@ func ServerSemVer(ctx context.Context, client *containerd.Client) (*semver.Versi
 	return sv, nil
 }
 
+func buildctlVersion() dockercompat.ComponentVersion {
+	buildctlBinary, err := buildkitutil.BuildctlBinary()
+	if err != nil {
+		logrus.Warnf("unable to determine buildkit version: %s", err.Error())
```

Comment:
```suggestion
		logrus.Warnf("unable to determine buildctl version: %s", err.Error())
```

(Same for other lines too)